src/esm_runscripts/prepcompute.py

```diff
@@ -171,10 +167,9 @@ def modify_namelists(config):
     # Load and modify namelists:
 
     if config["general"]["verbose"]:
-        print("\n" "- Setting up namelists for this run...")
+        logger.debug("\n" "- Setting up namelists for this run...")
         for index, model in enumerate(config["general"]["valid_model_names"]):
-            print(f'{index+1}) {config[model]["model"]}')
-        print()
+            logger.debug(f'{index+1}) {config[model]["model"]}')
```

Comment:
Don't you need to get rid of the `config["general"]["verbose"]`.

Reply:
Removed `config["general"]["verbose"]`